setup.py

```diff
@@ -74,9 +74,45 @@ def run(self):
         install.run(self)
 
 
+class LlvmliteClean(Command):
```

Comment:
I didn't see a clean command in setuptools, only in distutils. Looks like this is also not too concise https://bitbucket.org/carljm/python-distutils/src/48c42eeaee4410d76675b637bcd401b8919ff19a/command/clean.py?at=default&fileviewer=file-view-default But I agree, it would make sense to derive from it anyways. I basically had taken the next best receipe and adapted it to some extended needs.